### DISCUSSION TITLE
CHM help file

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#980] [Ideas]
CHM is offline, more organized and better than `read the docs` because everything is there, in the same CHM file. The CHM file could be extracted to put it online like this: https://pascalabc.net/downloads/pabcnethelp/index.htm, very handy for people on Linux since they can't read CHM natively.

**Comment by LADSoft** [comment]
i had chm files at one point but they are hard to maintain.   the MD files can be read offline with a suitable viewer (notepad ++ will do it in a pinch)....
